ReText/window.py

```diff
@@ -211,8 +211,8 @@ def __init__(self, parent=None):
 			trig=lambda: self.insertChars('*'))
 		self.actionUnderline = self.act(self.tr('Underline'), shct=QKeySequence.Underline,
 			trig=lambda: self.insertTag('u'))
-		self.usefulTags = ('a', 'big', 'center', 'img', 's', 'small', 'span',
-			'table', 'td', 'tr', 'u')
+		self.usefulTags = ('header', 'italic', 'bold', 'numbering', 'bullets', 'image', 'link',
```

Comment:
I'm trying to add a "stand alone" shortcut without lucky!
I found this example that should work with Qt4

    self.boldShortcut = QShortcut(QKeySequence.Bold, self, None, self.insertTag("bold"))

But it does not work in Qt5! Any idea?